src/components/common/LoginFacebook/LoginFacebook.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import FacebookLogin from "react-facebook-login";
+import cn from "classnames";
+import styles from "./LoginFacebook.module.scss";
+
+const appId = "473685456965260";
+
+type Props = {
+  buttonText: string;
+  className?: string;
+};
+
+function LoginFacebook(props: Props) {
+  const onCallback = (res) => {
+    console.log("[Facebook callback] currentUser:", res.profileObj);
+  };
+
+  return (
+    <div>
+      <FacebookLogin
+        appId="562118384400275"
+        autoLoad={true}
```

Comment:
I thiiink this makes it so that the facebook popup appears before actually clicking the button.